classes/class-gf-securesubmit.php

```diff
@@ -1423,11 +1440,6 @@ private function authorizeCC($feed, $submission_data, $form, $entry)
                 return $this->authorization_error(wp_sprintf('%s %s', $fraud_message, $issuerResponse));
                 //throw new HpsException(wp_sprintf('%s %s', $fraud_message, $issuerResponse));
             }
-            $response = $this->getSecureSubmitJsResponse();
-            $token = new HpsTokenData();
-            $token->tokenValue = ($response != null
-                ? $response->token_value
-                : '');
```

Comment:
Moving this into `buildCardHolder` and not renaming the method/variable is confusing from a maintenance perspective.

Reply:
updated